components/skeleton/index.zh-CN.md

```
@@ -0,0 +1,62 @@
+---
+category: Components
+type: 反馈
+title: Skeleton
+subtitle: 骨架屏
+---
+
+在需要等待加载内容的位置设置一个骨架屏，可以降低用户的认知负担，使加载过程更自然。
+
+### 规则
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)